src/Http/Http.Api.Abstractions/src/IFromBodyAttribute.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Http.Api.Abstractions
+{
+    /// <summary>
+    /// Interface marking attributes that specify a parameter or property should be bound using the request body.
+    /// </summary>
+    public interface IFromBodyAttribute
```

Comment:
nit: I would not use attribute here, but Metadata

Reply:
I just want to make sure I understand this correctly. Are you suggesting we rename this to `IFromBodyMetadata`?

Reply:
Yep, and all the similar interfaces that have Attribute suffix.